packages/radix-ui-themes/changelog.md

```diff
@@ -21,6 +21,7 @@
     - `gridRow`, `gridRowStart`, `gridRowEnd`
   - Deprecate `shrink` and `grow` props in favour of `flexShrink` and `flexGrow`. The older `shrink` and `grow` props will be removed in the next major release.
   - Update the type signature of the layout props so that code editor suggestions use just space scale values when possible. CSS keywords and other values such as `"auto"` or `"100vw"` are still available as manual string values.
+  - Fix an issue with responsive props when using a breakpoints object without the `initial` key would not apply the default prop value
```

Comment:
Had this fat bug on my list and realized I had accidentally fixed it with that arbitrary props PR

The bug was that components like `<Button size={{ xl: "2" }} />` would appear broken until `xl` and not use the default value